src/asm/output.c

```diff
@@ -541,26 +549,24 @@ static void checksectionoverflow(uint32_t delta_size)
  */
 void out_WriteObject(void)
 {
-	FILE *f;
+	FILE *f = NULL;
+	struct PatchSymbol *pSym;
+	struct Section *pSect;
 
 	addexports();
 
 	/* If no path specified, don't write file */
-	if (tzObjectname == NULL)
-		return;
+	if (tzObjectname != NULL) {
+		f = fopen(tzObjectname, "wb");
+		if (f == NULL)
+			fatalerror("Couldn't write file '%s'\n", tzObjectname);
 
-	f = fopen(tzObjectname, "wb");
-	if (f == NULL)
-		fatalerror("Couldn't write file '%s'\n", tzObjectname);
-
-	struct PatchSymbol *pSym;
-	struct Section *pSect;
+		fwrite(RGBDS_OBJECT_VERSION_STRING, 1,
+		strlen(RGBDS_OBJECT_VERSION_STRING), f);
```

Comment:
Could you indent the arguments so that they don't line up with `fwrite`?

Reply:
Yes, my bad.